src/algo/depth-to-rgb-calibration/optimizer.h

```diff
@@ -18,6 +18,7 @@ namespace librealsense {
 namespace algo {
 namespace depth_to_rgb_calibration {
 
+    const int DIRECTIONS = direction::deg_180;
```

Comment:
So this means we have 0==deg_0, 1==deg_45, 2==deg_90, and 3==deg_135?
If so, please make a comment to this effect.

But still: what sets this N_DIRECTIONS separate from direction enum? Maybe it should go alongside it, in calibration-types.h?